ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -121,6 +120,30 @@ class AlertsView extends React.Component {
     return pages;
   };
 
+  composeParams = (id, page, framework, status) => {
+    let params;
+    if (id) {
+      params = { id };
+    } else {
+      params = {
+        framework: framework.id,
+        page,
+      };
+    }
+
+    // created for UI purposes but not valid as API parameter
+    const doNotFilter = -1;
+    if (!id && summaryStatusMap[status] !== doNotFilter) {
+      params.status = summaryStatusMap[status];
+    }
+
+    if (params.framework === doNotFilter) {
```

Comment:
This if-conditional should mimic the previous one, for consistency reasons, because the `status` & `framework` filtering mechanics are very much the same.

Reply:
Changed it. Took a clearer aproach.